sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -175,7 +177,7 @@ case class HashAggregateExec(
   }
 
   // The variables used as aggregation buffer. Only used for aggregation without keys.
```

Comment:
nit: let's add a comment `Each aggregate function has one more more ExprCode to update its buffer slots`